dask/compatibility.py

```diff
@@ -77,6 +79,16 @@ def reraise(exc, tb=None):
     from urllib2 import urlopen
     from urlparse import urlparse, urlsplit
     from urllib import quote, unquote
+    try:
+        import pathlib2 as pathlib
+        _HAVE_PATHLIB = True
+    except ImportError:
+        try:
+            import pathlib
+            _HAVE_PATHLIB = True
+        except ImportError:
+            _HAVE_PATHLIB = False
```

Comment:
Instead of a separate `_HAVE_PATHLIB` variable I recommend setting `pathlib = False`

```python
try:
    import pathlib
except ImportError:
    pathlib = False
```

Then in use

```python
if pathlib:
    # do stuff with pathlib
```

Reply:
I like this pattern because it reduces the number of terms and because I think that I see it more often in related projects